### DISCUSSION TITLE
yari doesn't mean anything

### DISCUSSION THREAD

**Discussion by softorangetech1122** [mdn/mdn-community#203] [Content]
According to Wikipedia, the name "Yari" is a Japanese blade, and it is not a real word.
![Yari is a Japanese blade](https://user-images.githubusercontent.com/111031232/186545323-9b163568-25a9-4e16-a27f-3abc5c4df51c.png)
![Yari is not a word](https://user-images.githubusercontent.com/111031232/186545362-429b8a62-3460-46b2-99a1-39bd7e884643.png)

So why call the platform code storehouse `yari`? I suggest it to be called `platform-code`.

**Comment by teoli2003** [comment]
Thank you for your question and suggestion

This is a proper name. Many engines or tools have names. For example, the OS of Microsoft is called `Windows` and not `Microsoft-OS`. Car makers name cars: Porsche named its SUV Cayenne and not Porsche SUV, …

The previous engines powering MDN were MediaWiki, DekiWiki, and Kuma. This one is called Yari. The engine is not always the same, it changes over time.

It is a name.

Additionally, it doesn't need to be in the dictionary. It is better for it not to be in the English dictionary as it makes SEO and searches easier and more efficient.
Even you named your tool Codasove and not softaorangetech200-code-coverage.

That's the same: Mozilla decided to name the new engine Yari, replacing the old one, Kuma.

(Anyway, nobody will change its name now).

I hope this clarifies the naming of the tool.
